docs/specs/tripgo.openapi.yaml

```diff
@@ -5036,6 +5036,51 @@ components:
           $ref: '#/components/schemas/ArrayOfSimpleDataSourceAttribution'
         localCost:
           $ref: '#/components/schemas/LocalCost'
+        geofences:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: Unique identifier of geofence
+              type:
+                type: string
+                description: Type of geofence
+                enum:
+                - CIRCLE
+              trigger:
+                type: string
+                description: When to trigger geofence
+                enum:
+                - ENTER
+                - EXIT
+              center:
+                description: Center coordinate for circular geofences
+                $ref: '#/components/schemas/Cooordinate'
+              radius:
```

Comment:
would `radius` be optional due to the existence of `type` (if not circle)?

Reply:
or just radius from the center in general?

Reply:
Done. Marked them as required.

If we end up adding another type of geofence, we can use the `type` as a "discriminator", and then mark them as required only if `type == center`.